src/components/parameters/common/computing-type.ts

```diff
@@ -7,6 +7,8 @@
 
 export enum ComputingType {
     LOAD_FLOW = 'LOAD_FLOW',
```

Comment:
Why keeping `LOAD_FLOW` ComputingType ?

Reply:
You should add some cases management in the later :
`formatComputingTypeLabel` function then no ?